src/infrastructure/gateway/getPokemonByIdGateway.ts

```diff
@@ -0,0 +1,17 @@
+import { Pokemon } from "../../domain/model/IPokemon";
+import { IGetPokemonByIdGateway } from "../../domain/port/output/IGetPokemonByIdGateway";
+import PokemonApiToPokemonMapper from "./mapper/pokemonApiToPokemonMapper";
+import { PokemonApi } from "./model/IPokemonApi";
+import FetchClientUtil from "./util/FetchClientUtil";
+
+export class GetPokemonByIdGateway implements IGetPokemonByIdGateway {
+  
+    URL_GET_POKEMON = "https://pokeapi.co/api/v2/pokemon/";
```

Comment:
Lo mismo que me comento ignacio, esta constante deberia estar fuera de la clase y no usar el this.